scheduler.go

```diff
@@ -856,9 +888,38 @@ func (s *Scheduler) Weeks() *Scheduler {
 }
 
 // Month sets the unit with months
+func (s *Scheduler) Month() *Scheduler {
```

Comment:
Let's make the signature of Month() and Months() to allow an int day or int days to be passed with a variadic function for this.

```suggestion
func (s *Scheduler) Month(dayOfTheMonth ...int) *Scheduler {
```

With the variadic function we can then do nice things like: `Month(1, 2, 3)` and this would remove the need for the `On()` func entirely.

Reply:
Actually, my first solution was exactly same option but issue additional context changed my mind.  Done